tests/test_duckdb_pandas_tds_compare.py

```diff
@@ -19,16 +19,18 @@
 if DATABASE_FILEPATH.is_file():
     DATABASE_FILEPATH.unlink()
 
-TDS = TEEHRDatasetDB(DATABASE_FILEPATH)
+tds = TEEHRDatasetDB(DATABASE_FILEPATH)
 
 # Perform the join and insert into duckdb database
-TDS.insert_joined_timeseries(
+tds.insert_joined_timeseries(
     primary_filepath=PRIMARY_FILEPATH,
     secondary_filepath=SECONDARY_FILEPATH,
     crosswalk_filepath=CROSSWALK_FILEPATH,
     drop_added_fields=True,
 )
 
+# Join the attributes
+tds.join_attributes("/home/matt/repos/teehr/tests/data/test_study/geo/test_attr.parquet")
```

Comment:
Looks like the path is hardcoded to your local directory

Reply:
Good catch.  Will address.